packages/button/src/Button.js

```diff
@@ -42,83 +43,71 @@ type Props = {
   /** Display a primary button */
   primary?: boolean,
   /** Available sizes */
-  size?: 'small' | 'medium' | 'large',
+  size?: 'small' | 'medium' | 'large' | 'xlarge',
```

Comment:
Should we change this to `'sm' | 'md' | 'lg' | 'xl'`, so that all options are 2 letters (and more concise)? Or is that not worth doing because `'xxl'` would break the convention?